src/platformClient.js

```diff
@@ -0,0 +1,82 @@
+const axios = require('axios');
+const uuid = require('uuid');
+
+const invalidToken = 'Invalid token';
+
+class PlatformClient {
+  /**
+   * Constructor
+   * @param logger function
+   */
+  constructor(logger) {
```

Comment:
Should we rename this to `loggerFunc`, or alternatively use `logger.log` below?

In here, we could then do something like this:

```javascript
this.loggerFunc = (logger && logger.log) || console.log;
```

Reply:
I agree with the renaming and the default, but I don't want to assume that the `logger` has `log` method.